app/src/main/java/com/github/se/project/ui/lesson/EditRequestedLesson.kt

```diff
@@ -40,37 +42,38 @@ fun EditRequestedLessonScreen(
 
   val context = LocalContext.current
 
-  /**
-   * Lambda function to handle lesson updates. Updates the lesson details and navigates back to the
-   * home screen upon completion.
-   *
-   * @param lesson The updated lesson to be saved.
-   */
   val onConfirm = { lesson: Lesson ->
-    // Updates the lesson in the ViewModel and refreshes the user's lesson list
+    /**
+     * Lambda function to handle lesson updates. Updates the lesson details and navigates back to
+     * the home screen upon completion.
+     *
+     * @param lesson The updated lesson to be saved.
+     */
     lessonViewModel.updateLesson(
         lesson,
         onComplete = {
           lessonViewModel.getLessonsForStudent(profile.value!!.uid, onComplete = {})
-          Toast.makeText(context, "Lesson updated successfully", Toast.LENGTH_SHORT).show()
+          Toast.makeText(context, context.getString(R.string.lesson_updated), Toast.LENGTH_SHORT)
+              .show()
         })
 
     navigationActions.navigateTo(Screen.HOME)
   }
 
-  /**
-   * Lambda function to handle lesson deletion. Deletes the selected lesson and navigates back to
-   * the home screen upon completion.
-   *
-   * @param lesson The lesson to be deleted.
-   */
   val onDelete = { lesson: Lesson ->
+    /**
+     * Lambda function to handle lesson deletion. Deletes the selected lesson and navigates back to
+     * the home screen upon completion.
+     *
+     * @param lesson The lesson to be deleted.
+     */
```

Comment:
Same as before, I am not sure it is better to move the comment block inside the function.